File: 2st/Riyeon/1로 만들기.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        for(int i : num_list){
            while(i!=1){
                if(i%2==1) i--;
                i /= 2;
                answer ++;
            }
        }
        return answer;
    }
}

File: 3st/Hyeonseo/programmer/1로 만들기.py
def solution(num_list):
    answer = 0
    for n in num_list:
        while n > 1:  
            if n % 2 == 0:  
                n //= 2
            else:
                n = (n - 1) // 2
            answer += 1 
    return answer
d